helvetica-chimica-acta

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Helvetica Chimica Acta</title>
+    <id>http://www.zotero.org/styles/angewandte-chemie</id>
```

Comment:
1. change this line to match with your self link.
2. And I guess angewande-chemie was your template style. Please add that below the self link with rel="template".
3. add the .csl extension to your filename.